### ISSUE TITLE
NVDA did not read the voice results

### ISSUE THREAD

**Issue by amunger** [microsoft/vscode#203196]
Testing #203087

![Recording 2024-01-23 at 09 42 24](https://github.com/microsoft/vscode/assets/2019016/3b287dff-6634-43ab-bac0-4fd9d3ca1166)

**Comment by meganrogge** [comment]
@amunger did this always reproduce for you?

**Comment by amunger** [comment]
yes, I never got it to read the result.

**Comment by meganrogge** [comment]
do you receive other alerts around the workbench? for example, if you save a document manually, do you hear save?

**Comment by amunger** [comment]
I hear the queue but 'save' is not announced

**Comment by meganrogge** [comment]
I think you've got an issue with your NVDA configuration then. Can you pls try resetting that as I cannot repro?

**Comment by meganrogge** [comment]
Closing as it is working for me and I believe there's some issue with your setup

![image (27)](https://github.com/microsoft/vscode/assets/29464607/3d12073b-e158-4f34-92a5-67cf230e816d)
